tests/bitsandbytes/test_mixed_int8.py

```diff
@@ -146,6 +146,17 @@ def test_linear_are_8bit(self):
                 if name not in ["lm_head"] + T5PreTrainedModel._keep_in_fp32_modules:
                     self.assertTrue(module.weight.dtype == torch.int8)
 
+    def test_llm_skip(self):
+        r"""
+        A simple test to check if `llm_int8_skip_modules` works as expected
+        """
+        quantization_config = BitsAndBytesConfig(load_in_8bit=True, llm_int8_skip_modules=["classifier"])
+        seq_classification_model = AutoModelForSequenceClassification.from_pretrained(
+            "roberta-large-mnli", quantization_config=quantization_config
+        )
+        self.assertTrue(isinstance(seq_classification_model.classifier.dense, nn.Linear))
```

Comment:
Just for my own understanding (not a comment to address), here we're checking the layers of the classifier are `nn.Linear`. In `test_linear_are_8bit`, we check that the layers are `nn.Linear` too and that their dtype is `torch.int8` (I didn't know this was possible!). Are we certain that this means these layers are loaded in correctly? Do we need a dtype check on the weights?

Reply:
You are right, we also need a dtype check on the weights! `Linear8bitLt` has `nn.Linear` as a super class. Adding new tests!